File: QE-Python_selenium/qe-selenium-python/Activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/keyboard-events")

print(driver.title)

# Locate the heading element using the same XPath
keyboard = driver.find_element(By.XPATH, "/html/body/div/main/div/div/div/div/div[2]/h1")

# Pause for 3 seconds (like Thread.sleep in Java)
time.sleep(3)

# Print heading text
print(keyboard.text)

driver.quit()

File: QE-Java-Selenium/QE-selenium-java/javaselenium/src/main/java/javaselenium/demos/Activity9.java
package javaselenium.demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity9 {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/keyboard-events");
        System.out.println(driver.getTitle());
        WebElement keyboard=driver.findElement(By.xpath("/html/body/div/main/div/div/div/div/div[2]/h1"));
        Thread.sleep(3000);
        System.out.println(keyboard.getText());
        driver.quit();
    }}